pilot/cmd/pilot-agent/main.go

```diff
@@ -354,11 +354,11 @@ var (
 					LocalHostAddr: localHostAddr,
 					LocalPort:     stsPort,
 				}, tokenManager)
+				defer stsServer.Stop()
```

Comment:
If the **err != nil**，stsServer maybe nil？

Reply:
Sorry, didn't see this